Makefile

```diff
@@ -43,6 +44,10 @@ cluster/seed/smtp:
 cluster/seed/blobstorage:
 	oc apply -f ./deploy/crds/integreatly_v1alpha1_blobstorage_cr.yaml -n $(NAMESPACE)
 
+.PHONY: cluster/seed/redis
+cluster/seed/redis:
+	oc apply -f ./deploy/crds/integreatly_v1alpha1_redis_crd.yaml -n $(NAMESPACE)
```

Comment:
Should this be the cr rather that the CRD

Reply:
It should